packages/runtime/src/widgets/Search.tsx

```diff
@@ -75,10 +75,6 @@ export const Search: React.FC<SearchProps> = ({
   const onSelectAction = useEnsembleAction(onSelect);
 
   useEffect(() => {
-    if (isEmpty(namedData)) {
-      return;
-    }
-
```

Comment:
@evshi this needs to be remove because of, is there were no records found on specific query, then we need to reset the options list back to empty array.

For example,

https://github.com/EnsembleUI/ensemble-react/assets/31651612/8f846bd1-e44a-44c1-b5e1-3e0083ba1070

Reply:
isEmpty checks for `undefined`, `null` and empty array. You still need to check for null/undefined otherwise things will error out below.

Reply:
You are right, but `namedData` will never be null or undefined from the `useTemplateData` hook.

<img width="613" alt="image" src="https://github.com/EnsembleUI/ensemble-react/assets/31651612/28102ec5-9aee-4e89-953a-757dc2445874">